x-pack/auditbeat/module/system/host/host.go

```diff
@@ -322,16 +339,20 @@ func hostEvent(host *Host, eventType string, action eventAction) mb.Event {
 	event := mb.Event{
 		RootFields: common.MapStr{
 			"event": common.MapStr{
-				"kind":   eventType,
-				"action": action.String(),
+				"kind":     eventType,
+				"category": []string{"host"},
+				"type":     []string{action.Type()},
+				"action":   action.String(),
 			},
 			"message": hostMessage(host, action),
 		},
 		MetricSetFields: hostFields,
 	}
 
 	// Copy select host.* fields in case add_host_metadata is not configured.
-	hostTopLevel := common.MapStr{}
+	hostTopLevel := common.MapStr{
+		"name": host.Hostname,
```

Comment:
This is one of the quirks of the current libbeat pipeline setup. It adds `host.name` and the user can customize the name via setting `name: my_custom_name` in their config. So this override their custom name which is a breaking change. Run a quick test to see if you lose the custom name. If so just omit this change for now. We want to change this part libbeat for 8.0.

Reply:
just stripped it out